task/01-strings-tasks.js

```diff
@@ -99,7 +99,14 @@ function getFirstChar(value) {
  *   '\tHello, World! ' => 'Hello, World!'
  */
 function removeLeadingAndTrailingWhitespaces(value) {
-    throw new Error('Not implemented');
+    while (value.startsWith(' ') || value.startsWith('\t')) {
+        value = value.substr(1);
+    }
+    while (value.endsWith(' ') || value.endsWith('\t')) {
+        value = value.slice(0, -1);
+    }
+
```

Comment:
you can use a single regExp here or String.trim function

Reply:
Updated.